Seems i m still missing something 

```
/Users/builder/azdo/_work/1/s/src/DotNet/Dependencies/Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Workload.Mono.ToolChain.net6.Manifest-. No packages exist with this id in source(s): darc-pub-dotnet-emsdk-c3fc739, darc-pub-dotnet-runtime-1e1f688, dotnet-eng, dotnet-public, dotnet6, dotnet7, skiasharp [/Users/builder/azdo/_work/1/s/src/DotNet/DotNet.csproj]
/Users/builder/azdo/_work/1/s/src/DotNet/Dependencies/Workloads.csproj : error NU1101: Unable to find package Microsoft.NET.Workload.Mono.ToolChain.net7.Manifest-. No packages exist with this id in source(s): darc-pub-dotnet-emsdk-c3fc739, darc-pub-dotnet-runtime-1e1f688, dotnet-eng, dotnet-public, dotnet6, dotnet7, skiasharp [/Users/builder/azdo/_work/1/s/src/DotNet/DotNet.csproj]
```

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -79,6 +79,7 @@
     <DotNetVersionBand Condition=" '$(DotNetVersionBand)' == '' ">7.0.100</DotNetVersionBand>
     <DotNetSdkManifestsFolder>$(DotNetVersionBand)$([System.Text.RegularExpressions.Regex]::Match($(MicrosoftDotnetSdkInternalPackageVersion), `\-(preview|rc).\d+`))</DotNetSdkManifestsFolder>
     <DotNetSdkManifestsFolder>7.0.200$([System.Text.RegularExpressions.Regex]::Match($(MicrosoftDotnetSdkInternalPackageVersion), `\-(preview|rc).\d+`))</DotNetSdkManifestsFolder>
+    <DotNetMonoManifestVersionBand>$(DotNetVersionBand)</DotNetMonoManifestVersionBand>
     <DotNetMauiManifestVersionBand>$(DotNetVersionBand)</DotNetMauiManifestVersionBand>
     <DotNetEmscriptenManifestVersionBand>$(DotNetVersionBand)</DotNetEmscriptenManifestVersionBand>
     <DotNetAndroidManifestVersionBand>$(DotNetVersionBand)</DotNetAndroidManifestVersionBand>

